service/src/main/java/tds/config/repositories/impl/ClientTestPropertyQueryRepositoryImpl.java

```diff
@@ -0,0 +1,77 @@
+package tds.config.repositories.impl;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.util.Optional;
+
+import tds.config.ClientTestProperty;
+import tds.config.repositories.ClientTestPropertyQueryRepository;
+import tds.config.repositories.impl.mappers.ClientTestPropertyRowMapper;
+
+@Repository
+public class ClientTestPropertyQueryRepositoryImpl implements ClientTestPropertyQueryRepository {
+    private static final Logger LOG = LoggerFactory.getLogger(ConfigRepositoryImpl.class);
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public ClientTestPropertyQueryRepositoryImpl(DataSource dataSource) {
+        this.jdbcTemplate = new NamedParameterJdbcTemplate(dataSource);
+    }
+
+    @Override
+    public Optional<ClientTestProperty> findClientTestProperty(final String clientName, final String assessmentId) {
+        final SqlParameterSource parameters = new MapSqlParameterSource("clientName", clientName)
+            .addValue("assessmentId", assessmentId);
+
+        final String SQL =
+            "SELECT\n" +
+                "   clientname AS clientName,\n" +
+                "   testid AS assessmentId,\n" +
+                "   maxopportunities AS maxOpportunities,\n" +
+                "   prefetch AS prefetch,\n" +
+                "   isselectable AS isSelectable,\n" +
+                "   label AS label,\n" +
```

Comment:
having the alias be the same as the column name feels strange, but i'm guessing you are doing it for consistency? since almost all are being aliased.